src/app/appointments/NavBar.tsx

```diff
@@ -0,0 +1,178 @@
+"use client";
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Menu from "@mui/material/Menu";
+import MenuIcon from "@mui/icons-material/Menu";
+import Container from "@mui/material/Container";
+// import Avatar from "@mui/material/Avatar";
+
+import MenuItem from "@mui/material/MenuItem";
+
+import HealthAndSafetyIcon from "@mui/icons-material/HealthAndSafety";
+
+// const pages = ["Products", "Pricing", "Blog"];
```

Comment:
Never forget to remove comments. 
```suggestion

import MenuItem from "@mui/material/MenuItem";

import HealthAndSafetyIcon from "@mui/icons-material/HealthAndSafety";

```

![image](https://github.com/user-attachments/assets/61670cbf-c480-4da8-88a4-b0dfc545d827)